lib/actions/action-create-session.ts

```diff
@@ -1,10 +1,11 @@
 import * as assert from '@balena/jellyfish-assert';
 import type { ActionFile } from '@balena/jellyfish-plugin-base';
-import { v4 as uuidv4 } from 'uuid';
+import { TypeContract } from '@balena/jellyfish-types/build/core';
+import * as uuid from '@balena/jellyfish-uuid';
```

Comment:
We should stick with `uuid` instead of `@balena/jellyfish-uuid`. We archived `jellyfish-uuid` so we could reduce the amount of code we need to maintain.